pig-tests/src/org/partiql/pig/tests/generated/partiql-basic.kt

```diff
@@ -1730,6 +1729,16 @@ class PartiqlBasic private constructor() {
             override fun hashCode(): Int = myHashCode
         }
     
+        /** Converts instances of [PartiqlBasic.Projection] to any [T]. */
+        interface Converter<T> {
+            open fun convert(node: PartiqlBasic.Projection): T = when(node) {
+                is PartiqlBasic.Projection.ProjectList -> convertProjectList(node)
+                is PartiqlBasic.Projection.ProjectValue -> convertProjectValue(node)
+            }
+    
+            fun convertProjectList(node: PartiqlBasic.Projection.ProjectList): T
+            fun convertProjectValue(node: PartiqlBasic.Projection.ProjectValue): T
+        }
```

Comment:
Ack, my intention was to make this an abstract class--I may go back and do that still, but that shouldn't affect the rest of your review efforts.

Reply:
On second thought, I can't remember *why* I had intended to make this an abstract class and in that case, that `open` is redudant.